src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -204,11 +205,11 @@ public static Result canEntitySpawn(Mob entity, LevelAccessor world, double x, d
         return event.getResult();
     }
 
-    public static boolean canEntitySpawnSpawner(Mob entity, Level world, float x, float y, float z, BaseSpawner spawner)
+    public static boolean canEntitySpawnSpawner(SpawnData spawnData, Mob entity, Level world, float x, float y, float z, BaseSpawner spawner)
```

Comment:
I suggest this `Level` parameter be changed to type `LevelAccessor`, since all subsequent calls are of type `LevelAccessor`.  This probably should've been done a while ago, but since we're doing breaking changes we may as well do it now.